lightningd/chaintopology.c

```diff
@@ -515,7 +520,7 @@ static void json_feerates(struct command *cmd,
 static const struct json_command feerates_command = {
 	"feerates",
 	json_feerates,
-	"Add/query feerate estimates, either satoshi-per-kw ({style} sipa) or satoshi-per-kb ({style} bitcoind) for {urgent}, {normal} and {slow}."
+	"Query feerate estimates, either satoshi-per-kw ({style} sipa) or satoshi-per-kb ({style} bitcoind) for {urgent}, {normal} and {slow}."
```

Comment:
{urgent}, {normal} and {slow} here are not parameters any more
and the `withdraw_command` help (below) misses the {feerate} parameter

Reply:
Hmm, those fields are still *returned* though.  I'll clean it up to clarify (as a separate PR, since this has been acked and I'd like to merge).  Similar for the other two...